libs/ui-lib-tests/cypress/views/customOCPModal.ts

```diff
@@ -0,0 +1,33 @@
+export const customOCPModal = {
+  getCustomOCPModalId: () => {
+    return cy.get('#available-openshift-versions-modal');
+  },
+  inputVersionToSearch: (ocpVersion: string) => {
+    cy.get('#typeahead-select-input').clear().type(ocpVersion);
+  },
+  getSearchResponse: (searchResponse: string) => {
```

Comment:
```suggestion
  getSearchResponse: (searchResponse: string) => {
    return cy.get('#select-typeahead-listbox').within(() => {
      return cy.get('.pf-v5-c-menu__list-item').contains(searchResponse);
    });
  },
```
Then you don't need the function `selectFirstSearchResponse`, you could just do 
`customOCPModal.getSearchResponse(<VERSION_TO_SELECT>).click()`

Reply:
This change breaks the test of "No results found". I don't touch this.